src/hooks/useKnockoutExplorer.ts

```diff
@@ -101,40 +102,35 @@ async function configureFabric(): Promise<Explorer> {
           return;
         }
 
-        const data: FabricMessage = event.data?.data;
+        const data: FabricMessageV2 = event.data?.data;
         if (!data) {
           return;
         }
 
         switch (data.type) {
           case "initialize": {
-            const fabricDatabaseConnectionInfo: FabricDatabaseConnectionInfo = {
-              endpoint: data.message.endpoint,
-              databaseId: data.message.databaseId,
-              resourceTokens: data.message.resourceTokens as { [resourceId: string]: string },
-              resourceTokensTimestamp: data.message.resourceTokensTimestamp,
-            };
-            explorer = await createExplorerFabric(fabricDatabaseConnectionInfo);
+            const fabricVersion = data.version;
```

Comment:
This should be an array of versions so we can make Fabric support backward versions

Reply:
This is the version of the fabric client that is currently talking to DataExplorer. The array containing the supported versions is hardcoded in the respective clients.